Core/include/Acts/TrackFinding/CombinatorialKalmanFilter.hpp

```diff
@@ -309,7 +312,8 @@ class CombinatorialKalmanFilter {
         // splitting.
         // -> Call branch stopper to justify each branch
         // -> If there is non-outlier state, update stepper information
-        // 2) The surface is not in the measurement map but with material
+        // 2) The surface is not in the measurement map but with material or is
+        // an active surface
```

Comment:
Will this allow hole-search in the CKF ? That would be great

Reply:
I might be missing something but I think it was always the case that we could. But like for the fitter we would only found holes if the surface had material, that is no longer the case.

Reply:
I see. Before, I just assumed that the active surface always has material, which is not correct.